Mathlib/LinearAlgebra/QuadraticForm/Prod.lean

```diff
@@ -141,10 +143,10 @@ theorem anisotropic_of_prod {R} [OrderedCommRing R] [Module R M₁] [Module R M
   constructor
   · intro x hx
     refine' (h x 0 _).1
-    rw [hx, zero_add, map_zero]
+    rw [hx, zero_add, _root_.map_zero]
```

Comment:
Probably `map_zero` should be `protected` to avoid this issue. (in a pre-work CL)

Reply:
#14633